ember-flight-icons/addon/components/flight-icon.js

```diff
@@ -9,13 +9,13 @@ export default class FlightIconComponent extends Component {
   }
 
   /**
-   * Sets the fillColor for the SVG
+   * Sets the color for the SVG
    *
-   * @param fillColor {string}
+   * @param color {string}
    * @default 'currentColor'
    */
-  get fillColor() {
-    return this.args.fillColor ?? 'currentColor';
+  get color() {
+    return this.args.color ?? 'currentColor';
```

Comment:
This resolves the color TODO- color is set to `currentColor` as the default.

Reply:
i think the TODO is to add a test to check that / moved to https://github.com/hashicorp/flight/issues/54, so let's address that separately